fighthealthinsurance/models.py

```
@@ -2841,3 +2902,118 @@ class UsedDeleteToken(models.Model):
+    # Billing snapshot fields — encrypted to match Denial's posture so the
+    # audit table doesn't leak amounts that the parent record protects.
+    # Use the helpers (get_*_cents) for int round-trip; do NOT assign ints.
+    billed_amount_cents = EncryptedAmountField(blank=True, default="")
+    allowed_amount_cents = EncryptedAmountField(blank=True, default="")
+    paid_amount_cents = EncryptedAmountField(blank=True, default="")
+    created = models.DateTimeField(db_default=Now())
+
+    class Meta:
+        indexes = [
+            models.Index(fields=["denial", "created"], name="ucr_lookup_denial_idx"),
+        ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)